languages/metadata/languages.toml

```diff
@@ -0,0 +1,288 @@
+#:schema ../../schemas/generated/languages.json
+[ar]
+endonym = "اللغة العربية"
+locale = "ar-SA"
+name_en = "Standard Arabic"
+name_tok = "toki Alapi"
```

Comment:
maybe have these and `endonym` be fields under a `name` object?

Reply:
Trying this out. It felt a little weird at first, since the hypothetical correct place for `endonym` is actually `[id]`, but leaving it be seemed fine.